packages/lodestar/test/benchmarks/utils/index.ts

```diff
@@ -0,0 +1,29 @@
+import fs from "fs";
+
+export const createReportFile = (): string => {
+    const curDate: string = new Date().toISOString();
+    const dir: string = "./benchmark-reports/";
```

Comment:
Can you pull this up as a constant? `REPORT_DIR` or the like.

Reply:
yeah that makes sense